src/main/java/org/mifos/connector/common/channel/dto/Errors.java

```diff
@@ -0,0 +1,50 @@
+package org.mifos.connector.common.channel.dto;
+
+import org.mifos.connector.common.gsma.dto.ErrorParameter;
+
+import java.util.List;
+
+public class Errors {
+    public String errorCategory;
+    public String errorCode;
+    public String errorDescription;
+    public List<org.mifos.connector.common.gsma.dto.ErrorParameter> errorParameters = null;
+
+    public String getErrorCategory() {
```

Comment:
use Lombok @Getter and @Setter annotations instead of using setter getter method. This will reduce your code.